crates/re_renderer/shader/point_cloud.wgsl

```diff
@@ -104,6 +128,15 @@ fn fs_main(in: VertexOut) -> @location(0) Vec4 {
     return vec4(in.color.rgb * shading, coverage);
 }
 
+@fragment
+fn fs_main_picking_layer(in: VertexOut) -> @location(0) UVec4 {
+    let coverage = sphere_quad_coverage(in.world_position, in.radius, in.point_center);
+    if coverage == 0.0 {
```

Comment:
Shouldn't we cut off at the 0.5 alpha?

```suggestion
    if coverage <= 0.5 {
```

Reply:
I had this at first was a bit torn though because of the way the MSAA edge behaves then. But related to your inaccuracy comment, let's keep this scientific as possible and go with what you suggest here